istioctl/pkg/writer/ztunnel/configdump/configdump.go

```diff
@@ -49,14 +52,41 @@ func (c *ConfigWriter) PrintBootstrapDump(outputFormat string) error {
 
 // PrintSecretDump prints just the secret config dump to the ConfigWriter stdout
 func (c *ConfigWriter) PrintSecretDump(outputFormat string) error {
-	// TODO
+	if c.ztunnelDump == nil {
+		return fmt.Errorf("config writer has not been primed")
+	}
+	secretDump := c.ztunnelDump.Certificates
+	out, err := json.MarshalIndent(secretDump, "", "    ")
+	if err != nil {
+		return fmt.Errorf("failed to marshal workloads: %v", err)
+	}
+	if outputFormat == "yaml" {
+		if out, err = yaml.JSONToYAML(out); err != nil {
+			return err
+		}
+	}
+	fmt.Fprintln(c.Stdout, string(out))
 	return nil
 }
 
 // PrintSecretSummary prints a summary of dynamic active secrets from the config dump
 func (c *ConfigWriter) PrintSecretSummary() error {
-	// TODO
-	return nil
+	if c.ztunnelDump == nil {
+		return fmt.Errorf("config writer has not been primed")
+	}
+	secretDump := c.ztunnelDump.Certificates
+	w := new(tabwriter.Writer).Init(c.Stdout, 0, 8, 1, ' ', 0)
+	fmt.Fprintln(w, "RESOURCE NAME\tTYPE\tSTATUS\tVALID\tSERIAL NUMBER\tNOT AFTER\tNOT BEFORE")
+
+	for _, secret := range secretDump {
```

Comment:
Display serial number in hex format?

Reply:
will do

Reply:
@hanxiaop looking at the original description from https://github.com/istio/istio/issues/43765#issue-1609431544, is the format supposed to be alpha-numeric?  or should it contain colons (`8e:95:db:b0:4f:26:a3...`) ? `2d906bfcd420b8e3d...` vs `8e:95:db:b0:4f:26:a3...`.  after the fix in https://github.com/istio/istio/pull/43828, the format is `2d906bfcd420b8e3d...` - just wanted to verify that this is the desired behavior

Reply:
I think `2d906bfcd420b8e3d...` should be fine? Since this is the format shown when we use tools like openssl.